cognite/client/_api/data_modeling/instances.py

```diff
@@ -213,29 +214,29 @@ def retrieve(
         self,
         nodes: NodeId | Sequence[NodeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
         edges: EdgeId | Sequence[EdgeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
-        sources: list[ViewId] | ViewId | None = None,
+        sources: list[ViewIdentifier] | ViewIdentifier | None = None,  # list[ViewId] | ViewId | None = None,
         include_typing: bool = False,
     ) -> InstancesResult:
         """`Retrieve one or more instance by id(s). <https://docs.cognite.com/api/v1/#tag/Instances/operation/byExternalIdsInstances>`_
 
         Args:
             nodes (NodeId | Sequence[NodeId] | tuple[str, str] | Sequence[tuple[str, str]] | None): Node ids
             edges (EdgeId | Sequence[EdgeId] | tuple[str, str] | Sequence[tuple[str, str]] | None): Edge ids
-            sources (list[ViewId] | None): Retrieve properties from the listed - by reference - views.
+            sources (ViewId | Sequence[ViewId] | tuple[str, str] | Sequence[tuple[str, str]] | None): Retrieve properties from the listed - by reference - views.
```

Comment:
Should match signature.
```suggestion
            sources (ViewIdentifier | Sequence[ViewIdentifier] | None): Retrieve properties from the listed - by reference - views.
```